synfig-studio/src/gui/selectdraghelper.h

```diff
@@ -763,26 +771,44 @@ bool SelectDragHelper<T>::process_scroll_event(GdkEventScroll* event)
 		case GDK_SCROLL_UP:
 		case GDK_SCROLL_RIGHT: {
```

Comment:
Should we respect axis then?
If users has a device with a horizontal scroll, it should scroll horizontally without `Shift` :P

Reply:
* `Scroll`: move horizontally
* `Ctrl+Scroll`: zoom horizontally
* `Shift+Scroll`: move vertically
* `Ctrl+Shift+Scroll`: zoom vertically

This is your idea?

Reply:
For GDK_SCROLL_UP and GDK_SCROLL_DOWN cases, leave as PR does.

For GDK_SCROLL_RIGHT and GDK_SCROLL_LEFT, do as you proposed now (I actually thought only about the horizontal case, but I guess it's fine to `shift` mean axis-changing :) The Mirror tool uses the same concept)

Reply:
Done!

Reply:
Hmm… I just meant to separate the case `GDK_SCROLL_UP` from case `GDK_SCROLL_RIGHT`, and same for `GDK_SCROLL_DOWN` from `GDK_SCROLL_LEFT`.
The UP and DOWN would behave as you had implemented.
The RIGHT and LEFT would behave scrolling right/left respectively, but as horizontal zoom if with `ctrl` and vertically with `shift`

Reply:
For clarification:
```c++
		case GDK_SCROLL_UP:
		case GDK_SCROLL_RIGHT: {
		/// code
		}
```
would become
```c++
		case GDK_SCROLL_UP: {
		/// your code until your penultimate commit
		case GDK_SCROLL_RIGHT: {
		/// code similar to above, but swapping axis
		}
```

And analogous to to `DOWN`/`LEFT`